Dockerfile.rhel7

```diff
@@ -1,8 +1,8 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15 AS builder
 ADD . /usr/src/network-resources-injector
 RUN cd /usr/src/network-resources-injector && make
 
-FROM registry.ci.openshift.org/ocp/4.14:base
+FROM registry.ci.openshift.org/ocp/4.15:base-rhel9
```

Comment:
Need to bump in ocp-build-data https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.15/images/sriov-dp-admission-controller.yml#L16-L21

Reply:
https://github.com/openshift-eng/ocp-build-data/pull/3814

Reply:
Also need to bump https://github.com/zeeke/release/blob/master/ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml to avoid breaking quay.io images. see:

- https://github.com/openshift/release/pull/44739

Reply:
https://github.com/openshift/release/pull/44944